src/items/AppItem.js

```diff
@@ -16,6 +16,7 @@ export const GET_CONTEXT = 'GET_CONTEXT';
 export const GET_CONTEXT_SUCCEEDED = 'GET_CONTEXT_SUCCEEDED';
 
 const requestApiAccessToken = async ({ id, origin, app, apiHost }) => {
+  // eslint-disable-next-line no-undef
```

Comment:
Is this because of `fetch`? I think you should be able to tell ESLint that we're in a browser context and it should see `fetch`.